server/commands/screen_commands.c

```diff
@@ -559,7 +559,7 @@ screen_del_key_func(Client *c, int argc, char **argv)
 				*to++ = *from++;
 			}
 		}
-		to = '\0';	/* terminates the new keys string...*/
+		*to = '\0';	/* terminates the new keys string...*/
 	}
```

Comment:
The code has a logic to it, and I believe the proposed fix would make it work, but I agree that in the larger context, the code is basically unreachable. Best I can figure, the lookup has had the commands in front of this code commented out since at least 2002. I will update the pull request with a cleanup instead.